styleguide/components/Nav.css

```diff
@@ -1,5 +1,4 @@
 .nav {
-  composes: root from '../../components/Container/Container.css';
```

Comment:
Exactly. Worse than that, the `root` class I was composing from was completely empty.